geonode/maps/tests.py

```diff
@@ -21,7 +21,7 @@
 import logging
```

Comment:
`DEFAULT_CONTENT_TYPE` setting in this file is also depricated:
https://github.com/GeoNode/geonode/blob/c9672721ec7f974fb9465fa1ca796b6a5fa8b4e4/geonode/maps/tests.py#L662
source: https://docs.djangoproject.com/en/3.2/releases/3.0/#features-removed-in-3-0